### PR TITLE
Updated Text Editors lesson to include VSCode install steps

### PR DESCRIPTION
**Pull request by xandora**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [ ] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) and reference it with the `#` sign and the issue number.
 - [x] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

It's becoming obvious that just a link to the VSCode website isn't enough for many users. Update the text editor lesson to include instructions for installation of VSCode on both Ubuntu and macOS.

#### 2. Related Issue
N/A

### CODE REVIEW

File: foundations/installations/text_editors.md [hunk 1]
```diff
@@ -7,14 +7,87 @@ Rich text editors, such as Microsoft Word and Libre-Office Writer, are great for
...
+**Visual Studio Code**, or just VSCode as it's commonly referred to, is an excellent free code editor. It has outstanding add-on support and great Git integration. VSCode is the most popular code editor among Odin's students and moderators, so support is easy to find in the community.
 
 Which editor you use is generally a matter of preference, but for the purposes of this course, we are going to assume you're using VSCode, mainly because it's free, it's easy to use, and it works pretty much the same on every operating system. Keep in mind that means you will not able to get help if you are using a different text editor than VSCode for the curriculum.
 
 As a reminder, if you're using a **virtual machine**, you should install your text editor of choice **on your VM**. You're welcome to also install it on your host (i.e., your Windows main OS), but you'll want to be sure that you have this critical tool inside your VM.
```
Comment (01zulfi) [inline foundations/installations/text_editors.md:16]: The instructions are :fire:

Just one small thing. What do you think about this change?
```suggestion
As a reminder, if you're using a **virtual machine**, you should install VSCode **on your VM**. You're welcome to also install it on your host (i.e., your Windows main OS), but you'll want to be sure that you have this critical tool inside your VM.
```

Comment (xandora) [reply foundations/installations/text_editors.md]: Yup, I did consider changing that, but must have forgotten. :D

File: foundations/installations/text_editors.md [hunk 2]
```diff
@@ -7,13 +7,86 @@ Rich text editors, such as Microsoft Word and Libre-Office Writer, are great for
...
+<summary class="dropDown-header">Linux</summary>
+
+#### Step 1: Download VSCode
+
+   - Open your **Terminal**
+   - Run the following command to download the latest **VSCode** `.deb` package
+
+~~~bash
+curl -L -o code-latest.deb 'https://code.visualstudio.com/sha/download?build=stable&os=linux-deb-x64'
+~~~
```
Comment (01zulfi) [inline foundations/installations/text_editors.md:32]: Another small question I have.

 A quick google search says that we (TOP) have users first install curl in the [Installing Node.js lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/installing-node-js#installing-nvm). I don't think it's ideal if users run the curl command and its not installed on their systems. 
 
 Do you think it's worth adding an instruction to install curl?

Comment (xandora) [reply foundations/installations/text_editors.md:32]: We can change it to wget instead.

**Pull request merged**